mopidy_youtube/backend.py

```diff
@@ -82,13 +82,14 @@ def __init__(self, config, audio):
         youtube.Video.http_port = config["http"]["port"]
         youtube.Playlist.playlist_max_videos = config["youtube"]["playlist_max_videos"]
         youtube.musicapi_enabled = config["youtube"]["musicapi_enabled"]
-        youtube.musicapi_cookie = config["youtube"].get("musicapi_cookie", None)
-        youtube.musicapi_cookiefile = config["youtube"].get("musicapi_cookiefile", None)
-
-        if youtube.musicapi_cookie and youtube.musicapi_cookiefile:
-            raise ValueError("Only one of youtube/musicapi_cookie or youtube/musicapi_cookiefile can be used at one.")
-
-        youtube_music.own_channel_id = youtube.channel
+        if youtube.musicapi_enabled:
+            global youtube_music
+            from mopidy_youtube.apis import youtube_music
+            youtube.musicapi_cookie = config["youtube"].get("musicapi_cookie", None)
+            youtube.musicapi_cookiefile = config["youtube"].get("musicapi_cookiefile", None)
```

Comment:
No need to provide `None` default to `get(key)`, None is the default value by _default_.

Reply:
You technically don't even need to use `get()` on Mopidy config objects, they already have a default value.